src/components/Auth/Profile/tabs/OrderTab.jsx

```diff
@@ -1,218 +1,118 @@
-import React from "react";
+import { useEffect, useState } from "react";
+
+import { UserApi } from "../../../../api/auth/user";
 
 export default function OrderTab() {
+  const [orders, setOrders] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    const fetchUserOrders = async (id) => {
+      try {
+        const { data: userDetails } = await UserApi(id);
+        if (userDetails && userDetails.orders) {
+          setOrders(userDetails.orders);
+        } else {
+          setError("No orders found");
+        }
+      } catch (err) {
+        setError("Error fetching user data");
+        console.error(err);
+      } finally {
+        setLoading(false);
+      }
+    };
+
+    const token = localStorage.getItem("token");
+
+    if (!token) {
+      console.log("No token found");
+      return;
+    }
+
+    try {
+      const payload = token.split(".")[1];
+      const decodedPayload = atob(payload);
+      const decodedPayloadObject = JSON.parse(decodedPayload);
+
+      fetchUserOrders(decodedPayloadObject?.id);
+    } catch (error) {
+      console.error("Error decoding token:", error);
```

Comment:
Do not print the error 
```suggestion
        enqueueSnackbar( "Something went wrong, Please try again later!", { variant : "error" } );
```